docs/kcl-std/functions/std-sketch-rectangle.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "rectangle"
+subtitle: "Function in std::sketch"
+excerpt: "Sketch a rectangle."
+layout: manual
+---
+
+Sketch a rectangle.
+
+```kcl
+rectangle(
+  @sketchOrSurface: Sketch | Plane | Face,
+  width: number(Length),
+  height: number(Length),
+  center?: Point2d,
+  corner?: Point2d,
+): Sketch
+```
+
+
+
+### Arguments
+
+| Name | Type | Description | Required |
+|----------|------|-------------|----------|
+| `sketchOrSurface` | [`Sketch`](/docs/kcl-std/types/std-types-Sketch) or [`Plane`](/docs/kcl-std/types/std-types-Plane) or [`Face`](/docs/kcl-std/types/std-types-Face) | Sketch to extend, or plane or surface to sketch on. | Yes |
+| `width` | [`number(Length)`](/docs/kcl-std/types/std-types-number) | Rectangle's width along X axis. | Yes |
+| `height` | [`number(Length)`](/docs/kcl-std/types/std-types-number) | Rectangle's height along Y axis. | Yes |
+| `center` | [`Point2d`](/docs/kcl-std/types/std-types-Point2d) | The center of the circle. Incompatible with `corner`. | No |
+| `corner` | [`Point2d`](/docs/kcl-std/types/std-types-Point2d) | The corner of the circle. Incompatible with `center`. This will be the corner which is most negative on both X and Y axes. | No |
```

Comment:
And this will get auto generated right? Once replace circle with rectangle below?